naga/tests/naga/snapshots.rs

```diff
@@ -818,7 +818,9 @@ fn write_output_wgsl(
     input.write_output_file("wgsl", "wgsl", string);
 }
 
-#[cfg(feature = "wgsl-in")]
+// While we _can_ run this test under miri, it is extremely slow (>5 minutes),
+// and naga isn't the primary target for miri testing, so we disable it.
+#[cfg(all(feature = "wgsl-in", not(miri)))]
```

Comment:
This could also be expressed as `#[cfg_attr(miri, ignore)]`. That way, there is a record of tests being skipped. You may or may not find that worthwhile.